### ISSUE TITLE
Move wiki

### ISSUE THREAD

**Issue by Claycorp** [UnderMybrella/TiM#5]
Wikia is set up terribly please move your info off it to curseforge or put it on the Github wiki.
Much easier to use and allows for better navigation.

It also would be great if you could finish this as it took an extremely long time to figure out how the mod works as the documentation is poor. This isn't a mod that is really useable unless you document it as most people don't want ot spend hours looking for small bits of information.

**Comment by UnderMybrella** [comment]
I agree, Wikia is a pretty bad system, forgot that Github actually had wikis for repositories at the time...
I'll move most of the information over soon, although I'll keep the wikia up for part of the original purpose, which was to allow other people to share their configuration files, and I'm not sure if that can be done easily through Github's wiki (I could be wrong, I haven't really used the github wiki much in the past)
As for it not being finished, that's my fault again, sorry. I started it and then got caught up in a bunch of real life stuff, and forgot to add everything else. I'll keep this open until most of the information has been added.

**Comment by Claycorp** [comment]
I'm glad that you are fixing all this stuff :+1: More power to ya!
Also don't take it personal. Shit happens, people get busy. At least you are willing to admit that things need to get fixed!

(Hopefully my initial issues haven't came off too dickish.)

**Comment by UnderMybrella** [comment]
Na, they've been fine. But yeah, working on fixing all of the things, so a new version _should_ be out in the next day or two
